locksmith/src/routes/user.ts

```diff
@@ -72,6 +72,10 @@ router.get(
   '/:emailAddress/privatekey',
   userController.retrieveEncryptedPrivatekey
 )
+router.get(
+  '/:emailAddress/:selectedProvider/waas',
+  userController.retrieveWaasUuid
```

Comment:
Is there any authentication here? How can we make sure only the user who owns that email address is able connect??

Reply:
Can you please add the recpatcha middleware here? I am really worried about abuses on this API endpoint.